Sources/Library/InstantSearch.swift

```diff
@@ -105,6 +105,8 @@ import UIKit
     private lazy var viewModelFetcher: ViewModelFetcher = {
         return ViewModelFetcher()
     }()
+    
+    private static let instantSearchUserAgent = "InstantSearch iOS"
```

Comment:
How come there's no version here? It's usually `Algolia for Swift (X.X.X); InstantSearch iOS (X.X.X)`. The first part is added by IS core etc, but where's the version coming from